sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/implementation/ServiceBusAmqpConnection.java

```diff
@@ -44,5 +44,23 @@
      * @return A new or existing receive link that is connected to the given {@code entityPath}.
      */
     Mono<AmqpReceiveLink> createReceiveLink(String linkName, String entityPath, ReceiveMode receiveMode,
-        boolean isSession, String transferEntityPath, MessagingEntityType entityType);
+        String transferEntityPath, MessagingEntityType entityType);
+
+    /**
+     * Creates or gets an existing receive link. The same link is returned if there is an existing receive link with the
```

Comment:
Doc update: Creates a receive link for sessions. 
Since this is similar to the method above, I had mentioned to add the "SessionId" to the end of the parameter list as an overload. This would be easier to consume.